src/config.rs

```diff
@@ -22,19 +22,34 @@ pub enum ConfigurationError {
     MalformedSSORoleArn,
 }
 
-#[derive(Clone)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
_question_: sorry missed this one, is `Debug` required here or can we skip it?

Reply:
It's not needed, same for PartialEq. It was needed when I had the tests for asset, but right now we can remove it. Should I remove both ?

Reply:
Yes please :)